docs/apm/traces/view-and-investigate-traces.md

```diff
@@ -59,23 +59,49 @@ Queries above 7 days may be slower to load.
 
 The results are not automatically updated. If you want to refresh traces, click the refresh button on the top right corner of the page.<br/>![Refresh.png](/img/traces/Refresh.png)
 
-### Trace Duration Breakdown Chart
+### Trace Query Visualizations
 
-The Trace Duration Breakdown Chart helps you understand average trace durations in every time bucket as well as the amount of time each service contributed to the end-to-end duration. Use this chart to:
+The Trace Query Visualizations allow you to visualize the breakdown of where time was spent in traces for various services in your application. By aggregating traces, you can gain insights into anomalies and unexpected behaviors, leading to faster Time-to-Resolution (TTR) and a higher Return on Investment (ROI). It offers six charts to help you analyze and trace data efficiently. These include:
+* Trace duration
+* Error count
+* Error count
+* Span count
+* Span count
+* Trace duration as histogram
 
-* Quickly understand intermittent duration spikes or slowdowns.
-* Immediately spot the offending service by comparing CPC contribution by service.
+<img src={useBaseUrl('/img/traces/breakdown2.png')} alt="breakdown2.png" width="950px" height="500" />
 
-![breakdown](/img/traces/breakdown2.png)
+For each chart type, you can choose to view it as either a **timeseries** or **histogram** chart. Here is some more detailed information about each chart type:
+* **Trace duration**. It allows you to quickly identify services that are causing spikes or slowdowns in trace duration. It helps you understand the average trace duration for every time bucket, as well as the amount of time each service contributed to the end-to-end duration.
+* **Error count**. It helps you visualize the average number of errors per trace for certain time ranges. You can drill down to specific time ranges and identify the distribution of errors.
```

Comment:
```suggestion
* **Error count**. 
Timeseries: It helps you visualise the average number of errors per trace for selected time range.
Histogram: Provides an insight in how many traces were found with certain number of errors. For example if a backend fails in a specific way, it can always generate similar number of errors for traces from that period.
```